main.js

```diff
@@ -41,8 +41,8 @@ let maxStats = {
 };
 const firewalls = ["1", "2", "3"];
 const ocrApiKey = "XXX";
-const db = "https://raw.githubusercontent.com/snollygolly/sourceio-automation/master/db.json";
-let message = "papa bless, one love /r/javascript";
+const db = "https://raw.githubusercontent.com/bozoweed/sourceio-automation/master/db.json";
+let message = "papa bless, one love ";
```

Comment:
cause /r/javascript can seen by the target and it's spoted that is a bot so for be more human i have delete that